anabot/runtime/installation/hub/__init__.py

```diff
@@ -38,6 +38,9 @@ def _wait_for_depsolve(initial=True):
         filename = '/tmp/anaconda.log'
         waitline = '.*software_selection: The selection has been checked.*'
 
+    if is_distro_version_ge('rhel', 10):
```

Comment:
What about Fedora? The message was added in Fedora 35 (anaconda-35.14-1)

Reply:
I was not sure about fedora, thanks for the info. Condition for fedora was added.